cmd/asset-syncer/main.go

```diff
@@ -38,21 +47,19 @@ func main() {
 }
 
 func init() {
-	cmds := []*cobra.Command{syncCmd, deleteCmd}
+	rootCmd.PersistentFlags().StringVar(&databaseType, "database-type", "mongodb", "Database to use. Choice: mongodb, postgresql")
```

Comment:
persisten flags are used to allow these flags to be set either in the root or in the command